shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -28,7 +28,9 @@ FlatlandExternalViewEmbedder::FlatlandExternalViewEmbedder(
     std::shared_ptr<FlatlandConnection> flatland,
     std::shared_ptr<SurfaceProducer> surface_producer,
     bool intercept_all_input)
-    : flatland_(flatland), surface_producer_(surface_producer) {
+    : flatland_(flatland),
+      surface_producer_(surface_producer),
+      intercept_all_input_(intercept_all_input) {
```

Comment:
GfxExternalViewEmbedder seems to make use of input_interceptor_node_ in ctor as well. I think it might be the scenario of a flutter app that doesn't draw anything. I cant come up with a logical one, but it makes sense to mimic the behavior.

Reply:
Done